index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="css/project.css" rel="stylesheet">
+    <script src="./js/main.js" defer></script>
+    <title>Awesome Books</title>
+</head>
+
+<body>
+    <header class="header">
+        <h4 class="logo">Awesome Books</h4>
+        <ul class="nav">
+            <li><a class="list" href="#">List</a></li>
+            <li><a class="addBook" href="#">Add new</a></li>
+            <li><a class="contact" href="#">Contact</a></li>
+        </ul>
+    </header>
+    <span class="time"></span>
+
+    <section class="bookListSection">
+        <h1 class="pageTitle">Awesome Books</h1>
+        <div class="bookSection">
+        </div>
+    </section>
+    <section class="bookCreation bookCreationSection">
+        <h2 class="pageTitle">Add a new book</h2>
+        <input class="title" type="text" name="Book Title" placeholder="Title" required><br>
+        <input class="author" type="text" name="Author" placeholder="Author" required><br>
+        <button class="btn remove" type="submit">Add</button>
+    </section>
+    <section class="contactSection contactSectionNav">
+        <h2 class="pageTitle">Contact information</h2>
+        <p>Do have any questions or you just want to say "Hello"?<br>You can reach out to us!</p>
+        <ul class="contact-info">
+            <li>E-mail: oyelakinr37@gmail.com</li>
+            <li>Phone number: +2348134122452</li>
+            <li>Address: 18, Alaagba Street Paara Airport Area Ibadan, Oyo State, Nigeria</li>
+        </ul>
+    </section>
+    <footer class="footerSection header">
+        <h5 class="footer">&copy; 2022 Key. All Right Reserved | 🏗 by OyelakinG ✍✍</h5>
+    </footer>
```

Comment:
[OPTIONAL] Also, do you mind if I ask the reason why the name of your partner is not listed on either the contact section or the README file. From what I can tell he did indeed participate in the project. If there were any discrepancies between the both, remember that you can always reach student success either through the support button on the Microverse Dashboard or directly in Zendesk